spec/acceptance/mod_php_spec.rb

```diff
@@ -27,6 +27,11 @@ class { 'apache::mod::php': }
         describe file("#{apache_hash['mod_dir']}/php7.0.conf") do
           it { is_expected.to contain 'DirectoryIndex index.php' }
         end
+      elsif os[:family] == 'debian' && os[:release] =~ %r{10}
+        require 'pry'; binding.pry
```

Comment:
The CI is failing because of this line. I don’t know what it does but it doesn't seem required for other test cases, is it really needed here?